static/js/index.js

```diff
@@ -37,3 +38,27 @@ exports.aceInitialized = function(hook, context) {
   thisPlugin.authorsColors = authorsColorsUtils.init(editorInfo);
   editorInfo.ace_doInsertColors = _(doInsertColors).bind(context);
 };
+
+/*
+ * When the user reloads the pad, the etherpad replaces his color with one
+ * generated by etherpad itself. In order to keep using the teksto colors,
+ * we need this hook, which check if the teksto colors already was set,
+ * and if so, this hook replaces the etherpad color with the teksto color.
+ */
```

Comment:
talvez a gente pode deixar mais claro que esse cara só é usado nos refreshes da vida rs.
Algo por aí
> The author background color from TK is different from the one set by the Etherpad.
> We set this color for the first time on the module XXX, calling editorInfor.XXX. 
> The problem is because every time a user refreshes the editor page its background color 
> it's updated to the one set by Etherpad. So, the other pad's users will see a different color from the TK team color.
> To avoid that, we use this hook to force the user's TK background color.